### DISCUSSION TITLE
trying to commit my changes I got a package error

### DISCUSSION THREAD

**Discussion by jose4125** [vercel/turborepo#1741] [Help] [answered]
trying to commit my changes I'm getting the following error. not sure why
`package.json: json: cannot unmarshal array into Go struct field TurboJSON.turbo.Pipeline of type fs.pipelineJSON`

this is my turbo.json
```
{
  "pipeline": {
    "build": {
      "dependsOn": ["^build"],
      "outputs": ["dist/**", ".next/**"]
    },
    "lint": {
      "outputs": []
    },
    "dev": {
      "cache": false
    }
  }
}
```

**Comment by mehulkar** [comment]
Apologies for the late reply @jose4125 . If you're still running into this, can you open an issue with steps to reproduce, please ? I don't see anything wrong with your turbo.json

**Comment by anthonyshew** [comment, marked as answer]
This was a bug that that has since been resolved!
